README.md

```diff
@@ -1,40 +1,66 @@
 # Email alert frontend
 
-## Purpose
+A frontend for creating and managing email subscriptions.
 
-![Email Alert Frontend](docs/email-alert-frontend.png)
+## Overview
 
-A consistent frontend for displaying email alert signup pages.
+### Signup
 
-## Live examples
+This app provides two routes for signing up to email:
 
-- [gov.uk/government/policies/academies-and-free-schools/email-signup](https://www.gov.uk/government/policies/academies-and-free-schools/email-signup)
+- A legacy `/:base_path/email-signup` route [[example](https://www.gov.uk/foreign-travel-advice/canada/email-signup)].
+
+Each of these routes corresponds to a content item with an [`email_alert_signup` schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/email_alert_signup.jsonnet). At the time of writing, `/foreign-travel-advice/*` and `/service-manual/*` still use this route.
+
+- A `/email-signup?link=/:base_path` route [[example](https://www.gov.uk/email-signup/?topic=/money)].
```

Comment:
I might move this up above the legacy examples, and have Legacy as a sub-heading

Reply:
Sure, although I've gone with **bold** to avoid too many headings.